hw11.py

```diff
@@ -0,0 +1,53 @@
+import time
+
+
+def name_and_time(func):
+    print(f'You called function "{func.__name__}" on {time.ctime()}')
```

Comment:
This log should be inside the `inner_deco_func` to work only when a decorated function is called but not when the decorator initialized